### PR TITLE
Database Migration

### PR DESCRIPTION
**Pull request by gfx**
* Use Orma 2.2.0 to use `renameColumn()`
* Migration: renames `Session.speakerId` to `.speaker`, `.placeId` to `.place`, and `categoryId` to `category`

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java [hunk 1]
```diff
@@ -87,7 +90,16 @@ public CompositeSubscription provideCompositeSubscription() {
...
     @Provides
     public OrmaDatabase provideOrmaDatabase(Context context) {
-        return OrmaDatabase.builder(context).build();
+        return OrmaDatabase.builder(context)
+                .migrationStep(115, new ManualStepMigration.ChangeStep() {
+                    @Override
+                    public void change(@NonNull ManualStepMigration.Helper helper) {
+                        helper.renameColumn("Session", "placeId", "place");
+                        helper.renameColumn("Session", "speakerId", "speaker");
+                        helper.renameColumn("Session", "categoryId", "category");
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: Just question for Orma.
If I want to change column name, I have to use this manual migration, right?

Comment (gfx) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: Yes, because Orma can't handle renaming tables nor columns.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: Thanks :smiley:

**Comment by konifar** [comment]
LGTM! It is good sample for Orma migration. Great.

**Comment by gfx** [comment]
Right. I just want an actual usage of Orma migration.

File: app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java [hunk 2]
```diff
@@ -87,7 +90,16 @@ public CompositeSubscription provideCompositeSubscription() {
     @Singleton
     @Provides
     public OrmaDatabase provideOrmaDatabase(Context context) {
-        return OrmaDatabase.builder(context).build();
+        return OrmaDatabase.builder(context)
+                .migrationStep(115, new ManualStepMigration.ChangeStep() {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: If the next version would be `1.2.0`, this number should be `120`?

Comment (gfx) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: `115` will work well, but `120` is better for readability.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java]: Got it. Thanks!

**Pull request merged**